content/blog/2021-10-05.md

```diff
@@ -0,0 +1,177 @@
+---
+title: The XMPP Newsletter September 2021
+author: emus
+date: 2021-10-05
+categories: ['Newsletter']
+---
+
+Welcome to the XMPP Newsletter covering the month of September 2021.
+
+Many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, especially throughout the current situation, please consider to say thanks or help these projects!
+
+[Read this Newsletter via our RSS Feed](https://xmpp.org/feeds/all.atom.xml)!
+
+Interested in supporting the Newsletter team? Read more at the bottom.
+
+Other than that - enjoy reading!
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here (with some delay):
+
+- The French translation appears at [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+- The German translation appears at [anoxinon.de](https://anoxinon.de/blog/)
+- The Italian translation appears at [NicFab.it](https://www.nicfab.it/)
+
+Many thanks to the translators and their work! This is a great help to spread the news! Please join them in their work or start over with another language!
+
+## XSF Announcements
+The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP projects now! Please apply via Open Collective. For more information, see the [announcement blog post](https://xmpp.org/2021/09/the-xsf-as-a-fiscal-host/).
+
+Furthermore, the [xmpp.org](https://xmpp.org) website received an update. It's now built using [Hugo](https://gohugo.io/) (instead of Pelican) which reduces maintentance effort significantly. The new website is based on Bootstrap 5, and has been developed with simplicity in mind. We also made sure to make contributions as easy as possible. Building the website locally requires a minimum of dependencies, and is possible via Docker and Vagrant as well.
+
+## Events
+
+[XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours) - Also, checkout our new [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)!
+
+[Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): Monthly Meeting of XMPP Enthusiasts in Berlin - always 2nd Wednesday of the month.
+
+## Articles
+
+OpenPGP for XMPP (OX) is slowly getting client implementations, in a German blog post DebXWoody takes us step by step in [enabling it and using it in Profanity](https://mov.im/?blog/debxwoody%40movim.eu/profanity-und-openpgp-for-xmpp-ox-YRMREh).
```

Comment:
implementations. In a German blog post, DebXWoody walks us step by step through the process of enabling OX and using it in Profanity

Reply:
@wurstsalat3000 many thanks - can you push these? I'm not home till sunday